requirements-dev.txt

```diff
@@ -1,5 +1,6 @@
 black==24.4.2
 flake8==7.0.0
+marisa_trie==1.1.1
```

Comment:
Shouldn't `platformdirs` also be included?

Reply:
I guess @adbar just forgot it when adjusting the `requirements-dev.txt`. I just added it there as well.

Reply:
Yes, that was the issue, thanks.